samples/README.md

```diff
@@ -663,8 +653,7 @@ Your Thing's [Policy](https://docs.aws.amazon.com/iot/latest/developerguide/iot-
     {
       "Effect": "Allow",
       "Action": [
-        "iot:Receive",
-        "iot:Subscribe"
+        "iot:Receive"
```

Comment:
Why did we split this?

Reply:
`iot:Receive` uses `.../topic:/...` while `iot:Subscribe` needs `.../topicfilter:/...`. If you combine them together then the subscribe permissions are not given.